saltyrtc/server/server.py

```diff
@@ -45,7 +58,10 @@
 
 
 @asyncio.coroutine
-def serve(ssl_context, key, paths=None, host=None, port=8765, loop=None):
+def serve(
+        ssl_context, key, paths=None, host=None, port=8765, loop=None,
+        event_callbacks: Dict[Event, List[Coroutine]] = None
```

Comment:
No space before and after `=` (unless this changed in PEP8 for type hints? But that would be weird).